eng/Versions.props

```diff
@@ -137,7 +137,7 @@
     <optimizationPGOCoreCLRVersion>1.0.0-prerelease.22355.3</optimizationPGOCoreCLRVersion>
     <!-- Not auto-updated. -->
     <MicrosoftDiaSymReaderNativeVersion>16.9.0-beta1.21055.5</MicrosoftDiaSymReaderNativeVersion>
-    <SystemCommandLineVersion>2.0.0-beta3.22114.1</SystemCommandLineVersion>
+    <SystemCommandLineVersion>2.0.0-beta4.22355.1</SystemCommandLineVersion>
```

Comment:
The version is also listed here: https://github.com/dotnet/runtime/blob/main/eng/Version.Details.xml#L51-L54

Reply:
Repo build succeeded for me locally without updating Details.xml, perhaps we can add a target to ensure both files are in sync?

Captured commit hash on Ubuntu with:

```sh
$ apt install -y exiftool
$ exiftool ~/.nuget/packages/system.commandline/2.0.0-beta4.22355.1/lib/net6.0/System.CommandLine.dll \
     | grep "Product Version  "
```

Reply:
This should be getting update automatically via codeflow that keeps it in sync. dotnet-maestro bot should be automatically submitting a PR like this https://github.com/dotnet/runtime/pull/71900/files to update the system.commandline reference. It is not happening for some reason.